examples/react/wallets/web3modal.ts

```diff
@@ -69,5 +69,3 @@ export const web3Modal = createAppKit({
   coinbasePreference: "eoaOnly", // Smart accounts (Safe contract) not available on NEAR Protocol, only EOA.
   allWallets: "SHOW",
 });
-
-reconnect(wagmiAdapter.wagmiConfig);
```

Comment:
`ethereum wallets` need to be fixed (the "reconnect on reload page" is not working)